html-boilerplate/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<title>Tiffany's Story</title>
+	</head>
+	<body>
+		<section>
+			<h1>About</h1>
+			<p>I love to travel and help others book their travel. I was a mainframe developer for 15 years.</p>
+			<h2>Experience</h2>
```

Comment:
The instructions want us to wrap each section (About, Experience, Connect) in it's own separate <section> tag, so we just need to wrap each one up in that.